identifiers/country-ph/congressional_district.csv

```diff
@@ -252,4 +252,7 @@ ocd-division/country:ph/cd:zamboanga_del_norte_3rd,Zamboanga del Norte's 3rd,,
 ocd-division/country:ph/cd:zamboanga_del_sur_1st,Zamboanga del Sur's 1st,,
 ocd-division/country:ph/cd:zamboanga_del_sur_2nd,Zamboanga del Sur's 2nd,,
 ocd-division/country:ph/cd:zamboanga_sibugay_1st,Zamboanga Sibugay's 1st,,
-ocd-division/country:ph/cd:zamboanga_sibugay_2nd,Zamboanga Sibugay's 2nd,,
\ No newline at end of file
+ocd-division/country:ph/cd:zamboanga_sibugay_2nd,Zamboanga Sibugay's 2nd,,
+ocd-division/country:ph/cd:south_cotabato_3rd,South Cotabato's 3rd,2025-05,
+ocd-division/country:ph/cd:maguindanao_del_norte,Maguindanao del Norte,2025-05,

```

Comment:
The Maguindanao del Norte and del Sur have been represented in the House of Representatives since 2022. I would expect validFrom to be set to 2022.

Reply:
Hi @azuser we have updated valid from date of Maguindanao del Norte and del Sur to 2022.

Reply:
@azuser we have updated the full valid from date for Maguindanao del Norte and Maguindanao del Sur
and full valid through dates for Maguindanao's 1st and Maguindanao's 2nd.